docs/_data/sidebar.yml

```diff
@@ -20,20 +20,23 @@ options:
       - title: Index
         url: /docs/apidocs/compiler-plugin/arrow.meta.quotes/index.html
 
-      - title: classOrObject
-        url: /docs/apidocs/compiler-plugin/arrow.meta.quotes/class-or-object.html
+      - title: Quotes
+        url: /quotes
 
-      - title: func
-        url: /docs/apidocs/compiler-plugin/arrow.meta.quotes/func.html
+      - title: class
```

Comment:
This menu will be autogenerated based on what @franciscodr and I discussed on Thursday so you don't have to worry about adding each one of the items.

Reply:
@raulraja does it make sense to remove all these items from the side bar then, or to add them? I will adjust accordingly!